SDKv2 Examples/quickstart-kotlin/src/main/java/com/mappedin/quickstart_kotlin/MainActivity.kt

```diff
@@ -29,5 +35,34 @@ class MainActivity : AppCompatActivity() {
                 }
             })
         })
+
+        //Set an MiMapViewListener to run custom code on certain map events
+        mapView.setListener(object : MiMapViewListener {
+            override fun onTapNothing() {
+                //Called when a point on the map is tapped that isn't a MiSpace or MiOverlay
+            }
+
+            override fun didTapSpace(miSpace: MiSpace?): Boolean {
+                //Called when an MiSpace is tapped, return false to be called again if multiple MiSpaces were tapped
+                return false
+            }
+
+            override fun onTapCoordinates(latLng: LatLng) {
+                //Called when any point is tapped on the map with the LatLng coordinates
+            }
+
+            override fun didTapOverlay(miOverlay: MiOverlay): Boolean {
+                //Called when an MiOverlay is tapped, return false to be called again if multiple MiOverlays were tapped
+                return false
+            }
+
+            override fun onLevelChange(miLevel: MiLevel) {
```

Comment:
should we put the focus on method here as mentioned in the quickstart? or should the focus on part be mentioned in the level selection guide? it might make more sense there.

Reply:
Yea I'll move the focus part for the level selection guide and show the entire code here in the quickstart